### ISSUE TITLE
Checking for App Store / TestFlight environment broken?

### ISSUE THREAD

**Issue by kevinrenskers** [bitstadium/HockeySDK-iOS#205]
It used to be possible to see if the app was dowloaded from TestFlight rather then from the App Store by checking `[[NSBundle mainBundle] pathForResource:@"embedded" ofType:@"mobileprovision"]`. If nil then App Store, otherwise TestFlight.

However, this seems to longer work. Not sure if it's iOS 9, the new Xcode or new iTunes Connect, but something has changed..

This does work correctly:

```
BOOL isRunningTestFlightBeta = [[[[NSBundle mainBundle] appStoreReceiptURL] lastPathComponent] isEqualToString:@"sandboxReceipt"];
```

See http://stackoverflow.com/questions/26081543/how-to-tell-at-runtime-whether-an-ios-app-is-running-through-a-testflight-beta-i?lq=1.

**Comment by lumaxis** [comment]
Thanks @kevinrenskers for letting us know about this!
Just to clarify: You're saying that the current way of detecting the App Store also identifies TestFlight as a false positive, right?

**Comment by kevinrenskers** [comment]
Yeah. This is your code:

```
if (![[NSBundle mainBundle] pathForResource:@"embedded" ofType:@"mobileprovision"]) {
  _appStoreEnvironment = YES;
}
```

However, `[[NSBundle mainBundle] pathForResource:@"embedded" ofType:@"mobileprovision"]` now always seems to return nil, even when you are in fact in TestFlight environment.

**Comment by anlinde** [comment]
Looks like Apple changed the way they sign Testflight builds (again). We'll take a look, thanks!

**Comment by lumaxis** [comment]
Hey @kevinrenskers, we just released [version 3.8.3](https://github.com/bitstadium/HockeySDK-iOS/releases/tag/3.8.3) which should also fix your TestFlight detection issue. Be sure to check it out! :)
